src/style/themes/index.d.ts

```diff
@@ -0,0 +1,352 @@
+type Color = string;
```

Comment:
I think this file is not needed as we don't need exposed prop interfaces of `PortraitInitials` and `PotraitGravatar` components (where the `Theme` interface is imported and used)

Reply:
I have already commented on this one: https://github.com/Sage/carbon/pull/3816/files#r609512477